### PR TITLE
gws 0.1.7 (new formula)

### PR DESCRIPTION
**Pull request by frodeaa**

### CODE REVIEW

File: Library/Formula/gws.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Gws < Formula
+  homepage "https://streakycobra.github.io/gws/"
+  url "https://github.com/StreakyCobra/gws/archive/0.1.7.tar.gz"
+  version "0.1.7"
+  sha256 "36c25392f5cb93a39441a1973842249a62b605a2a7e369a040c78450683697d0"
+
+  def install
+      bin.install "src/gws"
+      bash_completion.install "completions/bash"
+      zsh_completion.install "completions/zsh"
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: 2-space indent, please.

File: Library/Formula/gws.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Gws < Formula
+  homepage "https://streakycobra.github.io/gws/"
+  url "https://github.com/StreakyCobra/gws/archive/0.1.7.tar.gz"
+  version "0.1.7"
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: Don't need this line.

File: Library/Formula/gws.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  sha256 "36c25392f5cb93a39441a1973842249a62b605a2a7e369a040c78450683697d0"
+
+  def install
+      bin.install "src/gws"
+      bash_completion.install "completions/bash"
+      zsh_completion.install "completions/zsh"
+  end
+
+  test do
+    system "#{bin}/gws"
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: This is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23243/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_gws/

Comment (mikemcquaid) [inline Library/Formula/gws.rb]: Test it locally with `brew test gws`

**Comment by tdsmith** [comment]
lgtm, will pull without objections

File: Library/Formula/gws.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  def install
+    bin.install "src/gws"
+
+    bash_completion.install "completions/bash"
+    zsh_completion.install "completions/zsh"
+  end
+
+  test do
+    output = `#{bin}/gws`.strip
+    assert_equal "Not in a workspace", output
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: Could we create a workspace?

Comment (frodeaa) [inline Library/Formula/gws.rb]: yes, but a workspace requires a local git repository to work on, making it more complex. I think the test is sufficient for testing the installation

Comment (mikemcquaid) [inline Library/Formula/gws.rb]: Several other tests do e.g. `git init`, `git add`, `git commit` to set up a workspace or you can use `HOMEBREW_REPOSITORY`

File: Library/Formula/gws.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  def install
+    bin.install "src/gws"
+
+    bash_completion.install "completions/bash"
+    zsh_completion.install "completions/zsh"
+  end
+
+  test do
+    system "git", "init", testpath/"project"
+    system "gws", "init", testpath
```
Comment (xu-cheng) [inline Library/Formula/gws.rb]: `system "#{bin}/gws", ...`

File: Library/Formula/gws.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    output = `#{bin}/gws status testpath`.strip
```
Comment (xu-cheng) [inline Library/Formula/gws.rb]: Please use `shell_ouput`. And shouldn't `testpath` be `#{testpath}`?

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/gws.rb [hunk 7]
```diff
@@ -0,0 +1,22 @@
+class Gws < Formula
+  homepage "https://streakycobra.github.io/gws/"
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: Sorry, we've recently added a `desc` (description) field. Would you mind adding one? After that this should be good to go.

File: Library/Formula/gws.rb [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  url "https://github.com/StreakyCobra/gws/archive/0.1.7.tar.gz"
+  sha256 "36c25392f5cb93a39441a1973842249a62b605a2a7e369a040c78450683697d0"
+
+  depends_on "bash"
```
Comment (mikemcquaid) [inline Library/Formula/gws.rb]: What does it depend on `bash` for? Does it not work with the system version?

Comment (frodeaa) [inline Library/Formula/gws.rb]: @mikemcquaid yes, it depends on bash because it needs support for associative arrays 0d90928

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**